nuxeo-pipeline-apb/templates/nuxeo-service.yml.j2

```diff
@@ -30,7 +30,7 @@ spec:
     applicationUrl: "{{ app_name }}-{{ item }}.apps.prod.nuxeo.io"
     nuxeoPackages: "{{ global_config.nuxeo_packages }}"
     nuxeoTemplates: "{{ global_config.nuxeo_templates }}"
-    nuxeoStudioProject:
+    nuxeoStudioProject: "{{ global_config.nuxeo_studio_project }}"
```

Comment:
Not sure this is useful as we are using the `connectExistingSecret` configuration.

Reply:
fixed